app/assets/stylesheets/scaffold.css

```diff
@@ -0,0 +1,80 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+}
+
+body, p, ol, ul, td {
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
```

Comment:
Do not use pixels for your line-height, make it relative to the font-size (ex: ```font-size: 1.5;```)